entrypoint-legacy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+cat >&2 <<-EOF
++------------------------------------------------------------------------------+
+| WARNING: loadimpact/k6 Docker image has been replaced by grafana/k6.         |
+|          THIS IMAGE IS DEPRECATED and its support will be discontinued after |
+|          Dec 31, 2023. Please update your scripts to use grafana/k6 to       |
+|          continue using the latest version of k6.                            |
++------------------------------------------------------------------------------+
+
+EOF
+
+k6.original $@
```

Comment:
This can be the same binary used in the `grafana/k6` image if we change the entrypoint, as I mentioned above.

But two more things here:

- Using the absolute path is safer, as otherwise this would run any `k6` binary on the `PATH`. Come to think of it, we should also change the entrypoint of the `grafana/k6` image to be `/usr/bin/k6`. Can you do that here or in another PR? :pray:
- Quoting `$@` is safer, as otherwise arguments with spaces would be interpreted as arguments to k6 itself.

  Consider `docker run --rm -i loadimpact/k6 run --tag 'some tag=some value' - <examples/thresholds.js`, which would fail with a cryptic `accepts 1 arg(s), received 3`.

So this should be:

```suggestion
/usr/bin/k6 "$@"
```

Reply:
I'd rather prefer to replace the `k6` binary to keep the entrypoint from the `release` step. Well spotted on quoting the args list 👍

Reply:
Actually this would affect the `k6 help` output:
```
Usage:
  k6.original [command]
     ~~~~~~~~
```
Let's go with your suggestions then 👍